C/plugins/storage/sqlite/plugin.cpp

```diff
@@ -40,8 +41,37 @@ const char *default_config = QUOTE({
 			"default" : "5",
 			"displayName" : "Pool Size",
 			"order" : "1"
-			}
-		});
+		},
+		"nReadingsPerDb" : {
+			"description" : "Number of readings tables per  database",
```

Comment:
"The number of readings tables in each database that is created"

Reply:
done